src/pip/_internal/cmdoptions.py

```diff
@@ -434,9 +434,9 @@ def only_binary():
     '-b', '--build', '--build-dir', '--build-directory',
     dest='build_dir',
     metavar='dir',
-    help='Directory to unpack packages into and build in. '
-        'Note that an initial build still takes place in a temporary directory. '
-        'The location of temporary directories can be controlled by setting '
+    help='Directory to unpack packages into and build in. Note that '
+        'an initial build still takes place in a temporary directory. The '
```

Comment:
Add a space in this indentation.

Reply:
yes i have updated the CI logs and have updated the help text accordingly

Reply:
Ill monitor those things, and will fix the issues if any, will notify you with a mention when things will be looking fine

Reply:
Sure! No hurries. :)

To run the linters locally, run tox -e lint-py3